osquery/core/CMakeLists.txt

```diff
@@ -6,22 +6,70 @@
 #  in the COPYING file in the root directory of this source tree).
 #  You may select, at your option, one of the above-listed licenses.
 
-file(GLOB OSQUERY_CORE_PLATFORM "${PROCESS_FAMILY}/*.cpp")
-file(GLOB OSQUERY_CORE_PLATFORM_TESTS "tests/${PROCESS_FAMILY}/*.cpp")
-
-if(APPLE)
-  set(OSQUERY_CORE_PLATFORM
-    ${OSQUERY_CORE_PLATFORM}
-    darwin/conversions.cpp
-    darwin/iokit.cpp
-  )
+ADD_OSQUERY_LIBRARY(TRUE osquery_core "")
```

Comment:
I plan to change every cmake file in project, but slowly. 
Side note: In this case, explicit is always better than implicit.
Main point is globing really hurts when more people woring on project, because it allows to add file to project without any modification to cmake file, which leads to issues when you pull  changes but cmake do not detect any modification and do not rebuild your make/xcode/qt project which in result lead to wierd issues. Moreover globbing really push you to use variables, and variables are also bad in modern cmake.